tests/sysprep_test.go

```diff
@@ -219,7 +219,7 @@ func insertProductKeyToAnswerFileTemplate(answerFileTemplate string) string {
 var getWindowsSysprepVMISpec = func() v1.VirtualMachineInstanceSpec {
 	gracePeriod := int64(0)
 	spinlocks := uint32(8191)
-	firmware := types.UID(windowsFirmware)
+	firmware := types.UID(tests.WindowsFirmware)
```

Comment:
nit: not sure here, maybe `libwindows.Firmware`? But also in the next iteration, maybe.

Reply:
Not sure about the need of a `libwindows` package. Will leave as is for the moment, and see in the future. Thank you